autoware_map_msgs/srv/PartialPointCloudMap.srv

```diff
@@ -0,0 +1,7 @@
+# Area query for map loading
+# (PCD grids that overlaps with the area is going to be loaded)
+AreaInfo area
+
+---
+# Newly loaded PCD maps with ID
+PointCloudMapWithID[] ids_to_remove
```

Comment:
I would like to get from native speakers regarding naming.
cc @mitsudome-r 
example: 
- clear_ids
- removal_ids

Reply:
I think it is typo.
It should contain a word that refers to the newly loaded Map.

Reply:
Oh, yes, thanks. Fixed https://github.com/autowarefoundation/autoware_msgs/pull/39/commits/ae7ea53556d1f81f8b5c5146afe08118b3538dcf